Dockerfiles/ci/command.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
I'm just curious, what is the purpose of this shell file? Since the name is command.sh, I don't know what the meaning is.

Reply:
I see it runs playbook, testing alm examples, then test env name, but what is the general meaning?

Reply:
This command.sh acts as the running command in Dockerfile. 
whenever the container is run. This command.sh triggers.